packages/nimble-tokens/NimbleTokens/NimbleTokens.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net5.0</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
@atmgrifter00 NimbleTokens was the only project still targeting .net 5, do you know if that is intentional / if there is any concern updating to .net 6?

Reply:
I don't believe there is a reason to stay on .NET 5.

Reply:
Good enough for me! thanks :)